src/app/components/coin/coin.component.html

```diff
@@ -0,0 +1,13 @@
+<mat-card class="coin-card">
+<img mat-card-sm-image [src]="coin.image" alt="coin logo" class="coin-logo">
+   <mat-card-subtitle class="coin-name-label">
+      Coin name:
+   </mat-card-subtitle>
+   <mat-card-title class="coin-name">
+      {{ coin.name }}
+   </mat-card-title>
+   <mat-card-content class="coin-details">
+      <p class="coin-current-price"><span>Current price: </span>$ {{ coin.current_price.toFixed(2) }}</p>
+      <p><span>Last update: </span>{{ formatDate(coin.last_updated) }}</p>
```

Comment:
Nu ai nevoie sa faci o metoda pentru Date, Angular are by default acest pipe. 
`{{ date_Value | date [ : format [ : timezone [ : locale ] ] ] }}`
In cazul tau: 
`{{ formatDate(coin.last_updated) }}` => `{{ coin.last_updated | date: 'short' }}`

https://www.angularjswiki.com/angular/angular-date-pipe-formatting-date-times-in-angular-with-examples/

Reply:
ce faain! mulțumesc, și pentru resursă! @vanceacorina97 😁